docs/cli-help.txt

```diff
@@ -13,27 +13,27 @@ cert. Major SUBCOMMANDS are:
   register             Perform tasks related to registering with the CA
   rollback             Rollback server configuration changes made during install
   config_changes       Show changes made to server config during installation
+  update_symlinks      Update cert symlinks based on renewal config file
   plugins              Display information about installed plugins
+  certificates         Display information about certs configured with Certbot
```

Comment:
Yes; I did an update of that file to show how this PR was changing the output of `certbot --help all`; `rename` is missing because that landed after I did the manual update; we usually update `docs/` in real time but maybe there have been one or two exceptions in the past?